### PR TITLE
Pull request as suggested by Team Leader to master

### PR DESCRIPTION
**Pull request by AdrianHarenczyk**
Added little functionality to disable at start button "Start". When button "Losuj" is pressed, "Start" button enables. When this button is pushed, a new stage is created which in next phases will contain board on which players can acctualy play and shoot at enemys fleet.

### CODE REVIEW

File: src/main/java/placement/view/GUIGameScreen.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+/**
+ * The entry point of GUI package. Sets up a window, loads components.
+ */
+public class GUIGameScreen extends Application {
+    private static final int SCENE_WIDTH = 1200;
+    private static final int SCENE_HEIGHT = 900;
+    @Override
```
Comment (boczkas) [inline src/main/java/placement/view/GUIGameScreen.java:15]: \n

File: src/main/java/placement/view/GUIGameScreen.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    /**
+     * main method - actual entry point of the application
+     * @param args - console arguments used by FX
+     */
+    public static void main(String[] args) {
+        launch(args);
+    }
+}
```
Comment (boczkas) [inline src/main/java/placement/view/GUIGameScreen.java:31]: \n

File: src/main/java/placement/view/GUIGameScreenController.java [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+
+/**
+ * this Controller has aim to connect user interface with functions of the Battleship game.
+ */
+public class GUIGameScreenController {
+
+
+
+}
```
Comment (JaroslawZaczyk) [inline src/main/java/placement/view/GUIGameScreenController.java:11]: empty class?

**Review by boczkas — APPROVED** [review]
:+1:

**Review by JaroslawZaczyk — APPROVED** [review]

**Pull request merged**